bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -160,6 +160,13 @@ def get_inflevel(var, inflevel):
     return keepbits
 
 
+def _jl_bitround(X, keepbits):
+    Main.X = X
+    Main.keepbits = keepbits
+    # wrapping BitInformation.round!($X, int) # what does the $ mean?
```

Comment:
The `$` is only for benchmarking with `@btime` from BenchmarkTools.jl and escapes any calculations outside of the actual function call via string interpolation

Reply:
thanks